stdlib/urllib/request.pyi

```diff
@@ -234,7 +234,7 @@ class ftpwrapper:  # undocumented
     def file_close(self) -> None: ...
     def init(self) -> None: ...
     def real_close(self) -> None: ...
-    def retrfile(self, file: str, type: str) -> None: ...
+    def retrfile(self, file: str, type: str) -> Tuple[addclosehook, int]: ...
```

Comment:
This doesn't seem right. Instead of literally putting `addclosehook` here, this should be whatever it returns.

Reply:
Nevermind, it's a class, not an instance

Reply:
`addclosehook` is a class not method ;-)

Reply:
I think you thought it was a function because it begins in lowercase. Even Python core developers don't follow the rules. Lol